plugins/experimental/slice/server.cc

```diff
@@ -268,8 +268,8 @@ logSliceError(char const *const message, Data const *const data, HttpHeader cons
   crange.toStringClosed(normstr, &normlen);
```

Comment:
Please mitigate the reference to `SLICER_MIME_FIELD_INFO` on line 262 above.

Reply:
missed this sry